library/Fission/Web/User/UpdateExchangeKeys.hs

```diff
@@ -0,0 +1,33 @@
+module Fission.Web.User.UpdateExchangeKeys
+  ( API
+  , server
+  ) where
+
+import           Servant
+
+import           Fission.Prelude
+import           Fission.Authorization
+ 
+import qualified Fission.Web.Error as Web.Error
+
+import qualified Crypto.PubKey.RSA as RSA
+import qualified Fission.User      as User
+
+type API
+  =  Summary "Update Public Exchange Keys"
+  :> Description "Set currently authenticated user's root list of public exchange keys"
+  :> ReqBody '[JSON] [RSA.PublicKey]
+  :> Patch   '[PlainText, OctetStream, JSON] NoContent
```

Comment:
I think this is actually a `PUT` request, no? We’re replacing the entire record, not updating a portion of it. (Unless I misunderstand how it works — totally possible!)

Reply:
ah yup totally right :+1: